torch_frame/data/multi_nested_tensor.py

```diff
@@ -235,21 +244,27 @@ def _row_narrow(self, start: int, length: int) -> 'MultiNestedTensor':
         offset = self.offset[start * self.num_cols:end * self.num_cols + 1]
         values = self.values[offset[0]:offset[-1]]
         offset = offset - offset[0]
-        return MultiNestedTensor(num_rows=end - start, num_cols=self.num_cols,
-                                 values=values, offset=offset)
-
-    def _col_narrow(self, start: int, length: int) -> 'MultiNestedTensor':
+        return MultiNestedTensor(
+            num_rows=end - start,
+            num_cols=self.num_cols,
+            values=values,
+            offset=offset,
+        )
+
+    def _col_narrow(self, start: int, length: int) -> "MultiNestedTensor":
         r"""Helper function called by :obj:`narrow`."""
         assert start >= 0
         assert length > 0
         end = start + length
         if start == 0:
             assert end < self.num_cols
-            offset_mat = (self.offset[:-1].reshape(
-                self.num_rows, self.num_cols)[:, start:end + 1])
+            offset_mat = self.offset[:-1].reshape(self.num_rows,
```

Comment:
Formatting is off. I recommend to split into two commands.

Reply:
Fixed!